client/src/API/Constants.ts

```diff
@@ -0,0 +1,11 @@
+export const URL = "http://localhost:5000";
+
```

Comment:
@Solshuk and @bretpeters3n, here are the interfaces and shapes of the objects you will be passing to the backend through the methods that @bycho91 defined.